core/vm/contracts.go

```diff
@@ -79,6 +83,10 @@ var PrecompiledContractsIstanbul = map[common.Address]PrecompiledContract{
 
 	common.BytesToAddress([]byte{100}): &tmHeaderValidate{},
 	common.BytesToAddress([]byte{101}): &iavlMerkleProofValidate{},
+
```

Comment:
Let's use `Hilbert` as the hardfork name.

Reply:
@guagualvcha Changed this with other fixes. Please help review the latest.